Tests/UnitTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -465,14 +466,27 @@ private extension OfferingsManagerTests {
     enum MockData {
         static let anyAppUserID = ""
 
+        @DefaultDecodable.EmptyDictionary
+        static var metadata: [String: AnyDecodable] = [
+          "int": 5,
+          "double": 5.5,
+          "boolean": true,
+          "string": "five",
+          "array": ["five"],
+          "dictionary": [
+              "string": "five"
+          ]
+        ]
+
         static let anyBackendOfferingsResponse: OfferingsResponse = .init(
             currentOfferingId: "base",
             offerings: [
                 .init(identifier: "base",
                       description: "This is the base offering",
                       packages: [
                         .init(identifier: "$rc_monthly", platformProductIdentifier: "monthly_freetrial")
-                      ])
+                      ],
+                      metadata: _metadata)
```

Comment:
Nit per our style guide:
```suggestion
                      metadata: Self.metadata)
```